qiskit/pulse/instructions/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The ``instruction`` module holds the various ``Instruction`` s which are supported by
+Qiskit Pulse. Instructions each contain operands unique to that instruction, but all have an
```

Comment:
```suggestion
Qiskit Pulse. Instructions each contain operands unique to that instruction and have a defined
```

Reply:
duration and channels are also operands, aren't they?

Reply:
duration isn't I see now, but channels still is, I think

Reply:
Updeated this 👍